.github/workflows/ci.yml

```diff
@@ -41,8 +41,7 @@ jobs:
         with:
           node-version: 20.x
       - run: yarn workspaces focus hono-middleware @hono/${{ matrix.package }}
-      - run: yarn workspaces foreach --topological --recursive --from @hono/${{ matrix.package }} run build
-      - run: yarn workspace @hono/${{ matrix.package }} publint
+      - run: yarn workspaces foreach --topological --recursive --from @hono/${{ matrix.package }} run publint
```

Comment:
Was running into an issue where `tsup` was referencing `dist/*.d.ts` files from the previous `build` step when running `publint` 😩